src/components/TableForm/TableForms.js

```diff
@@ -14,18 +14,33 @@ import ConfirmDialog from "../ConfirmDialog"
 import TimerIcon from '@material-ui/icons/Timer';
 import EventIcon from '@material-ui/icons/Event';
 import { changeTileColor, time} from '../../components/calender/Calender'
+import { relativeTimeRounding } from 'moment'
+import { blue } from '@material-ui/core/colors';
+import './TableForm.scss';
+
 const useStyles = makeStyles(theme =>({
     pageContent: {
-        margin: theme.spacing(5),
-        padding: theme.spacing(3)
+    display: 'flex',
+    flexWrap: 'wrap',
+    width: '100%',
+        
+        
     },
     searchInput: {
-        width: '75%'
+        width: '550px',
```

Comment:
Do you think we should use 'relative', not absolute?

Reply:
fixed